www/vorj00/ukol_07/assets/js/main.js

```diff
@@ -0,0 +1,59 @@
+var points = 0;
+var cities = ['Barcelona', 'Dortmund', 'Madrid', 'Turin', 'Prague', 'New York', 'Paris', 'London', 'Beijing', 'Moscow'];
+var firstSelect = null;
+
+cities = cities.concat(cities);
+cities.sort(function() { return .5 - Math.random(); });
+
+function editPoints(change){
+    if((change > 0) || (points > 0)){
+        points+=change;
+    } else {
+        points = 0;
+    }
+}
+
+$(document).ready(function(){
+    const pexesoPoints = '#js-points';
+    const pexesoBoard = '#js-pexeso';
+    const pexesoCard = '.js-playable';
+
+    $(pexesoPoints).text(points);
+
+    $.each(cities, function(index, value){
+        $(pexesoBoard).append(`<div data-id="${index}" class="pexeso__card pexeso__card--playing js-playable"></div>`)
+    });
+
+    $(pexesoCard).on("click", function(){
+        let currentCard = $(this);
+        let currentCardId = currentCard.data("id");
+
+        currentCard.addClass('pexeso__card--selected').removeClass('pexeso__card--playing');
+        currentCard.text(cities[currentCardId]);
+
+        setTimeout(function(){
```

Comment:
setTimeout dava smysl jen tehdy pri otoceni nespravne dvojice karet, pak se to za sekundu otoci zpatky, aby uzivatel jeste stihl zapamatovat kartu